strings/names/last.txt

```diff
@@ -15,7 +15,6 @@ Anderson
 Andreev
 Ann
 Archibald
-Armstrong
```

Comment:
armstrong is a pretty common last name, and it has a pretty obvious meaning. hence its use in many popular characters